extensions/wikia/TrackingOptIn/js/trackingOptIn.js

```diff
@@ -44,9 +60,54 @@ define('wikia.trackingOptIn', [
 		window.Wikia.consentQueue.push(callback);
 	}
 
+	function geoRequiresTrackingConsent() {
+		return geoRequiresConsent;
+	}
+
+	function getPrebidGlobalVendorList() {
+		var request = new XMLHttpRequest();
+
+		log('Requesting IAB Global Vendor List', log.levels.debug, logGroup);
+
+		request.open('GET', prebidVendorListUrl, false);
+		request.send(null);
+
+		return JSON.parse(request.responseText);
+	}
+
+	function getPrebidConsentString(optIn) {
+		var cookie = cookies.get(prebidConsentStringCookie);
+
+		if (cookie) {
```

Comment:
Should we reset cookie once user change decision about opt in?

Reply:
Right, I will reset cookie if optIn value will be different than the one saved in consent string.